### DISCUSSION TITLE
Is there a way to select text field on double tab?

### DISCUSSION THREAD

**Discussion by brunodmn** [danvick/flutter_form_builder#1236] [Q&A]
## Environment

**7.7.0**

## Describe your question
I willing to know if its possible to select all text within a form field with a double tap, something like [this](https://stackoverflow.com/questions/72962932/select-text-in-textfield-on-double-click-in-flutter-windows-app).

I am able to do the selection on the field, but could not attribute it to the field, my code:
```
(...)
GestureDetector(
                onDoubleTap: (() {
                  final String email =
                      controller.formKey.currentState!.fields['email']!.value;
                TextSelection(baseOffset: 0, extentOffset: email.length);
                }),
(..)
```

**Comment by deandreamatias** [comment]
@brunodmn This still an issue?
